<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="connectEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>431</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="realtimeBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="readbackBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="inquireBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="connectAct"/>
    <addaction name="saveAct"/>
    <addaction name="refreshAct"/>
    <addaction name="closeAct"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connectAct"/>
   <addaction name="saveAct"/>
   <addaction name="refreshAct"/>
   <addaction name="closeAct"/>
  </widget>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="refreshAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
   <property name="toolTip">
    <string>refresh</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="closeAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
   <property name="toolTip">
    <string>close</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="connectAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/open device.png</normaloff>:/open device.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>connect</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="writeAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/write.png</normaloff>:/write.png</iconset>
   </property>
   <property name="text">
    <string>write</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
